arangod/StorageEngine/HotBackupFeature.cpp

```diff
@@ -330,5 +485,9 @@ bool HotBackupFeature::cancelled(std::string const& transferId) const {
   return false;
 }
 
+void HotBackupFeature::removeAllArangoSearchData() {
+  ::removeAllArangoSearchData();
```

Comment:
it seems having delegating method is redundant, why don't to implement it here

Reply:
Agreed, will do the change.